mods/creative/inventory.lua

```diff
@@ -33,16 +38,18 @@ function creative.init_creative_inventory(player)
 		end,
 	}, player_name)
 
-	creative.update_creative_inventory(player_name, minetest.registered_items)
+	return player_inventory[player_name]
 end
 
+-- Fills a player's creative inventory with the items listed by tab_content.
+--   calls init_creative_inventory if the inventory does not already exist
+-- @param player_name The name of the player
+-- @param tab_content A key-value list of item names to item defs
 function creative.update_creative_inventory(player_name, tab_content)
```

Comment:
would be good to change this to just `player` - however could require backwards compat code (if it's used)